utility/serverrcon.js

```diff
@@ -90,6 +91,35 @@ class ServerRcon {
     }
   }
 
+  /**
+   * 
+   * Checks if the server is up to date via a steam API call.
+   * @function 
+   */
+  async isServerUpToDate() {
+    try {
+      if (process.env.NODE_ENV === "test") {
+        return false;
+      }
+      // Get server version.
+      let serverResponse = await this.execute("version");
+      let serverVersion = serverResponse.match(/(?<=version ).*(?= \[)/);
+      // Call steam API to check if the version is the latest.
+      let response = await fetch(`https://api.steampowered.com/ISteamApps/UpToDateCheck/v0001/?appid=730&version=${serverVersion}&format=json`);
```

Comment:
I know this is being picky, but would you mind changing this to be in line with all other text replacement? i.e. `await fetch("https://api.steampowered.com/ISteamApps/UpToDateCheck/v0001/?appid=730&version="+serverVersion"+&format=json")` Something to format/refactor in the future, but for now I'd like everything to be consistent :)